SIT.Manager/ViewModels/Play/CharacterSummaryViewModel.cs

```diff
@@ -161,4 +169,23 @@ private async Task Play()
             await errorDialog.ShowAsync();
         }
     }
+
+    private async Task Logout()
+    {
+        AkiCharacter? character = _connectedServer.Characters.FirstOrDefault(x => x.Username == Profile.Username);
```

Comment:
We already keep an `AkiCharacter` reference in the class that you could use if I'm not mistaken

Reply:
Your right, I even missed ReSharper telling me that my new object was hiding an existing one DOH!  I'll get that sorted (and tested) shortly.

Reply:
Just pushed a commit to resolve this issue.